test/unit-tests/function/matrix/kron.test.js

```diff
@@ -29,13 +29,12 @@ describe('kron', function () {
   })
 
   it('should calculate product for empty 2D Arrays', function () {
-    assert.deepStrictEqual(math.kron([[]], [[]]), [[]])
+    assert.deepStrictEqual(math.kron([[]], [[]]), [])
```

Comment:
Since the inputs are 2d arrays, the output should stay a 2d array `[[]]` like it was before.